Classes/Netlogix/Cors/Service/CorsService.php

```diff
@@ -141,17 +121,21 @@ public function sendHeaders() {
 		} else {
 			$this->denyRequest();
 		}
+
+		if (Bootstrap::getEnvironmentConfigurationSetting('REQUEST_METHOD') === 'OPTIONS') {
+			exit;
```

Comment:
Können wir das auf solche OPTIONS-Requests beschränken, die einen "Access-Controll-Request-Method"-Header oder einen "Access-Controll-Request-Headers"-Header haben?
Wenn es sich eindeutig um einen CORS-Preflight handelt ist das exit in Ordnung. Es kann aber noch andere OPTIONS-Requests geben. Zum Beispiel könnte ein OPTIONS-Request auf eine listAction Metainforationen über die verwendbaren Argumente für Filter, Sortierung und Paginierung erzeugen.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#The_HTTP_request_headers